### DISCUSSION TITLE
False positive-parrot images

### DISCUSSION THREAD

**Discussion by Crockan** [infinitered/nsfwjs#707] [Model Training]
When using Mobilenet V2 4.2mb Model, the following images show false positives
[Parrot courtship dance](https://i.imgur.com/5QDpmTG.png) - Identified as Porn - 61.18%
[Parrot sleeping](https://i.imgur.com/cdsTryl.png) - Identified as Porn - 59.64%
[Parrot eating](https://i.imgur.com/LtR0GFZ.png) - Identified as Porn - 68.27%
[Parrot moving at high speed](https://i.imgur.com/6cCgtie.png) - Identified as Porn - 95.78%
[Banana](https://i.imgur.com/E4OAJri.png) - Identified as Porn - 95.21%
[Lovebird](https://i.imgur.com/pkEW2xe.png) - Identified as Porn - 61.15%
[Parrot's claws](https://i.imgur.com/gBPDw93.png) - Identified as Porn - 69.66%

Seems to have a strange false positives rate for parrots?

**Comment by Crockan** [comment]
A few additions:
[Parrot calling](https://i.imgur.com/czro6qC.png) - Identified as Porn - 70.59%
[Overlooking a parrot](https://i.imgur.com/bTvAEvF.png) - Identified as Porn - 91.37%
[Parrot](https://i.imgur.com/7QBFg1R.png) - Identified as Porn - 69.18%
[Parrot shakes its head](https://i.imgur.com/DaygdF6.png) - Identified as Porn - 51.20%
[More Parrot](https://i.imgur.com/oFxJvSa.png) - Identified as Porn - 70.51%

**Comment by GantMan** [comment]
That is strange!   How does the 90MB model perform?

**Comment by Crockan** [comment]
> That is strange! How does the 90MB model perform?

![chrome_mc5UthqZJD](https://user-images.githubusercontent.com/45941794/108090236-5b6a0b80-70b5-11eb-9dc0-35a1fdb22319.png)

The situation doesn't look promising either
I'll get more pictures to test it out

**Comment by GantMan** [comment]
What a crazy result!   We'll have to add parrot pictures to the training data next round.
